yt_dlp/extractor/fancode.py

```diff
@@ -12,27 +12,29 @@
 class FancodeVodIE(InfoExtractor):
     IE_NAME = 'fancode:vod'
 
-    _VALID_URL = r'https?://(?:www\.)?fancode\.com/video/(?P<id>[0-9]+)\b'
+    _VALID_URL = r'https?://(?:www\.)?fancode\.com(?:/[a-zA-Z0-9].*)?/videos?/(?:(?:[^/])*?/)?(?P<id>[0-9]+)\b'
 
     _TESTS = [{
         'url': 'https://fancode.com/video/15043/match-preview-pbks-vs-mi',
+        'only_matching': True
+    }, {
```

Comment:
Why is this changed to only_matching?

Reply:
They no longer seem to use brightcove anymore, so video id is diffferent now.